test/e2e-tests.sh

```diff
@@ -52,7 +52,7 @@ go_test_e2e -timeout=30m \
   ./test/conformance/... \
   ./test/e2e \
   ${parallelism} \
-  "--resolvabledomain=$(use_resolvable_domain)" || failed=1
+  "--resolvabledomain=$(use_resolvable_domain)" "--https=$(use_https)" || failed=1
```

Comment:
How does this work with 
```
    --https)
      readonly HTTPS=1
      return 1
      ;;
```
?

It looks to me that specifying `--https` will enable HTTPS no matter the value of `use_https`.
Am I missing something?

Reply:
Yep you are right!!